nixos/modules/services/audio/snapserver.nix

```
@@ -211,6 +212,15 @@ in
         '';
       };
 
+      extraConfig = lib.mkOption {
+        type = lib.types.str;
+        default = "";
+        example = "--logging.filter debug";
```

```suggestion
        type = with lib.types; listOf str;
        default = [ ];
        example = [ "--logging.filter debug" ];
```

Better to ensure proper tokenization.